ngraph/frontend/onnx_import/src/editor/editor.cpp

```diff
@@ -232,3 +242,41 @@ void onnx_import::ONNXModelEditor::set_input_shapes(
         }
     }
 }
+
+void onnx_import::ONNXModelEditor::cut_graph_fragment(const std::vector<InputEdge>& inputs,
+                                                      const std::vector<InputEdge>& outputs)
```

Comment:
You can use here `OutputEdge` alias

Reply:
Good catch, it was supposed to be a strong type alias and that way this method wouldn't even compile. I'll update the code to address it.